### ISSUE TITLE
what is the katago net used in UEC?

### ISSUE THREAD

**Issue by l1t1** [lightvector/KataGo#452]
http://entcog.c.ooco.jp/entcog/new_uec/list.html

**Comment by lightvector** [comment]
Just the standard one at the time on katagotraining.org listed as "strongest confident", which was s708.

**Comment by l1t1** [comment]
and the machine spec?

**Comment by Friday9i** [comment]
IIRC, it was 8 A100
By the way, how many visits per second did you approximately get, @lightvector please, something like 25K visits/second?

**Comment by lightvector** [comment]
Yes, just AWS's standard 8xA100 instance. It varied a lot depending on the phase of the game and how much transpositions there were. Maybe like 35k per second was common.
